flink-ai-flow/lib/airflow/airflow/migrations/versions/d47076ca1426_add_event_based_scheduler_support.py

```diff
@@ -113,6 +113,15 @@ def upgrade():
         op.create_index('message_queue_time', 'message', ['queue_time'], unique=False)
         op.create_index('message_state', 'message', ['state'], unique=False)
 
+    if 'event_progress' not in tables:
+        op.create_table(
+            'event_progress',
+            sa.Column('scheduling_job_id', sa.Integer(), nullable=False),
+            sa.Column('last_event_time', sa.BigInteger(), nullable=True),
```

Comment:
Because the `event_progress` has the `last_event_version`,  the `last_event_time` is unnecessary.